lib/orangelight/illiad_account.rb

```diff
@@ -2,38 +2,39 @@
 
 require 'faraday'
 require 'faraday-cookie_jar'
+module Orangelight
+  class IlliadAccount
+    def initialize(patron)
+      @barcode = patron['barcode']
+      @last_name = patron['last_name']
+      @patron_id = patron['patron_id']
+      @netid = patron['netid']
+      @illiad_api_key = Requests.config["illiad_api_key"]
+      @illiad_api_base = Requests.config["illiad_api_base"]
+    end
 
-class IlliadAccount
-  def initialize(patron)
-    @barcode = patron['barcode']
-    @last_name = patron['last_name']
-    @patron_id = patron['patron_id']
-    @netid = patron['netid']
-    @illiad_api_key = Requests.config["illiad_api_key"]
-    @illiad_api_base = Requests.config["illiad_api_base"]
-  end
-
-  def verify_user
-    begin
-      response = conn.get do |req|
-        req.url "/ILLiadWebPlatform/Users/#{@netid}"
-        req.headers['Accept'] = 'application/json'
-        req.headers['ApiKey'] = @illiad_api_key
+    def verify_user
+      begin
+        response = conn.get do |req|
+          req.url "/ILLiadWebPlatform/Users/#{@netid}"
+          req.headers['Accept'] = 'application/json'
+          req.headers['ApiKey'] = @illiad_api_key
+        end
+      rescue Faraday::ConnectionFailed
+        Rails.logger.info("Unable to Connect to #{@illiad_api_base}")
+        return false
       end
-    rescue Faraday::ConnectionFailed
-      Rails.logger.info("Unable to Connect to #{@illiad_api_base}")
-      return false
+      response.success?
```

Comment:
@maxkadel did you mean to move this?

Reply:
Everything is just indented one so the git compare is funky. It hasn't actually moved as far as I can tell.